cmd/cli/util_test.go

```diff
@@ -3,14 +3,20 @@ package main
 import (
 	"context"
 	"fmt"
+	"sort"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/runtime"
+
+	corev1 "k8s.io/api/core/v1"
+
```

Comment:
nit: I think the newlines can be removed as these are all external imports so can be grouped together